doc/build_apps/example_rpc_api.rst

```diff
@@ -2,7 +2,6 @@ Example app RPC API
 ===================
 
 .. openapi:: ../schemas/app_openapi.json
-   :examples:
```

Comment:
Is there a decent auto-renderer from TypeSpec already?

Reply:
I don't know, but don't think it's worth investigating at the moment: we only have a TypeSpec definition of the governance API (so not `/app` or `/node`), and we've not even got a copy of that in our repo at the moment (because the tooling was too unstable for us to consistently/automatically re-generate OpenAPI from it). Maybe once TypeSpec reaches 1.0 we should revisit this, see if its worth migrating all of our builtin APIs to be TypeSpec-first, iff it gives us nicer tooling integration (eg - into a doc generator).

FWIW I think the fact that our generated OpenAPI always stays in-sync with the implementation is a big win, and investment in improving the detail of what is generated (eg - adding explicit example responses) would probably make it play nicer with existing tools.